int32[] bone_ids
unity_robotics_demo_msgs/PosRot[] bone_poses
